src/ontology/nbo-edit.owl

```diff
@@ -1211,19 +1413,6 @@ SubClassOf(obo:NBO_0000009 obo:NBO_0000644)
 
 # Class: obo:NBO_0000010 (reproductive behavior)
 
-AnnotationAssertion(Annotation(obo:IAO_0000119 "Immelmann, K., & Beer, C. (1989). A dictionary of ethology. Harvard University Press.") Annotation(oboInOwl:hasDbXref "ISBN:0674205065") obo:IAO_0000115 obo:NBO_0000010 "A behavior process that tends to increase, “the propagation of new individuals”.")
-AnnotationAssertion(obo:IAO_0000116 obo:NBO_0000010 "Disjointing parental behavior follows original NBO structural intent despite many experts.")
-AnnotationAssertion(terms:contributor obo:NBO_0000010 <https://orcid.org/0000-0002-2061-091X>)
-AnnotationAssertion(terms:contributor obo:NBO_0000010 <https://orcid.org/0000-0002-4277-2151>)
-AnnotationAssertion(terms:date obo:NBO_0000010 "2011-04-14T01:03:39Z"^^xsd:date)
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:NBO_0000010 "reproduction")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:NBO_0000010 "reproductive behaviour")
-AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0000010 "behavior_ontology")
-AnnotationAssertion(oboInOwl:inSubset obo:NBO_0000010 <http://purl.obolibrary.org/obo/nbo#do_not_annotate>)
-AnnotationAssertion(oboInOwl:inSubset obo:NBO_0000010 <http://purl.obolibrary.org/obo/nbo#ethology_module>)
-AnnotationAssertion(rdfs:comment obo:NBO_0000010 "PRIOR DEFINITION: Behavior directly related to the production of offspring.")
-AnnotationAssertion(rdfs:label obo:NBO_0000010 "reproductive behavior")
-AnnotationAssertion(<http://www.w3.org/2004/02/skos/core#exactMatch> obo:NBO_0000010 obo:GO_0019098)
```

Comment:
Seems something wen wrong here. Is the term mentioned in the template?

Reply:
No - it was part of the Upper level ROBOT template that was recently merged. It shouldn't be being edited and this may also explain part of the error discussed in line 1160 above.

Reply:
This seems to have been a template bug.